src/Webhooks/Webhook.cs

```diff
@@ -5,13 +5,19 @@
 using System.Threading.Tasks;
 using Newtonsoft.Json;
 using UnityEngine.Networking;
+using UnityEngine;
+using BrokeProtocol.Managers;
+using BrokeProtocol.API;
+using System.Net;
+using System.Collections.Specialized;
 
 namespace Webhooks
 {
     [JsonObject]
-    public class Webhook
+    public class Webhook : ManagerEvents
```

Comment:
Is there a reason for this class to inherit from ManagerEvents?

Reply:
Yes because otherwise we can't access to SvManager

Reply:
You can still access it since you are setting it on the Core instance